client/orm/types.go

```diff
@@ -235,6 +235,8 @@ type Inserter interface {
 
 // QuerySeter query seter
 type QuerySeter interface {
+	// add query context for querySeter
+	WithContext(context.Context) QuerySeter
```

Comment:
As mentioned before, we don't need this method.

Reply:
Same as the above comment.